Updated RDS documentation to include SSL connection required for PostgreSQL version 15+ (#2417)
18F/cg-site: _docs/services/relational-database.md

- 14
- 15

> *\*Note: For PostgreSQL versions 15+ to connect to your database an SSL connection is required. As such you will need to configure your client library you use to connect to your database to use an SSL connection.*

Reviewer: ```suggestion
*\*Note: **[For PostgreSQL versions 15+ , using SSL is required when connecting to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.*
```

> \*Note: **[For PostgreSQL versions 15+ , using SSL is required when connecting to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.

Reviewer: ```suggestion
*\*Note: [**For PostgreSQL versions 15+ , using SSL is required when connecting to your database**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.*
```

> *\*Note: **[For PostgreSQL versions 15+ , using SSL is required when connecting to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.*

Reviewer: ```suggestion
\*Note: **[For PostgreSQL versions 15+ , using SSL is required when connecting to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.
```

> *\*Note: [**For PostgreSQL versions 15+ , using SSL is required when connecting to your database**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring)**. As such, you will need to configure your client library to use SSL when connecting to your database.*

Reviewer: ```suggestion
*\*Note: [**For PostgreSQL versions 15+ , using SSL is required when connecting to your database**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html#PostgreSQL.Concepts.General.SSL.Requiring). As such, you will need to configure your client library to use SSL when connecting to your database.*
```
